### PR TITLE
Include extra args in AccessDenied exceptions

### PR DESCRIPTION
**Pull request by phallguy**
Completes #172

### CODE REVIEW

**Comment by blackst0ne** [comment]
What's the status of this PR?

File: CHANGELOG.rdoc [hunk 1]
```diff
@@ -1,5 +1,12 @@
 Develop
 
+1.10.1 (January 27th, 2015)
+
+* Complete cancancan#172 - Include extra args in AccessDenied exception.
+
+1.10.0 (August 28, 2014)
+
+* Complete cancancan#115 - Specify authorization action for parent resources.
```
Comment (coorasse) [inline CHANGELOG.rdoc]: Please separate the two features in two different PR.

File: spec/cancan/ability_spec.rb [hunk 2]
```diff
@@ -376,6 +376,7 @@ class Container < Hash; end
       expect(e.message).to eq("Access denied!")
       expect(e.action).to eq(:read)
       expect(e.subject).to eq(:foo)
+      expect(e.extra_args.first).to eq(1)
```
Comment (coorasse) [inline spec/cancan/ability_spec.rb]: please add some more tests: check what is the content of `extra_args` in different cases (also when there is no extra_args)

**Comment by coorasse** [comment]
I'd be happy to include these features but I need two different PR. Thanks for your work

**Comment by phallguy** [comment]
It's been 2 years since I submitted this PR. Unfortunately we've moved on from cancan and I don't really have time to invest in splitting this into separate pieces.
